cli/save.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// saveDescription is used to describe save command in detail and auto generate command doc.
+var saveDescription = "save an image to a tar archive."
+
+// SaveCommand use to implement 'save' command.
+type SaveCommand struct {
+	baseCommand
+	output string
+}
+
+// Init initialize save command.
+func (save *SaveCommand) Init(c *Cli) {
+	save.cli = c
+	save.cmd = &cobra.Command{
+		Use:   "save [OPTIONS] IMAGE [IMAGE...]",
+		Short: "Save an image to a tar archive or STDOUT",
+		Long:  saveDescription,
+		Args:  cobra.ExactArgs(1),
```

Comment:
I think this part is incompatible with `save [OPTIONS] IMAGE [IMAGE...]`. please make sure the input args number. @xiechengsheng